### PR TITLE
Fix for issue 1028

### PR DESCRIPTION
**Pull request by laurieaw63**
Patch default service account for non-kind clusters with registry credentials.
Set modulesNamespace in blueprint testcases.

Signed-off-by: laurieaw laurieaw@us.ibm.com

### CODE REVIEW

File: manager/controllers/app/blueprint_controller_test.go [hunk 1]
```diff
@@ -24,7 +24,8 @@ func deployBlueprint(namespace string, shouldSucceed bool) {
 
 	// Set the correct namespace
 	blueprint.SetNamespace(namespace)
-	fmt.Printf("Blueprint controller unit test - blueprint namespace: %s\n", namespace)
+	blueprint.Spec.ModulesNamespace = namespace
```
Comment (shlomitk1) [inline manager/controllers/app/blueprint_controller_test.go:27]: Use  `utils.GetDefaultModulesNamespace() `here. The `namespace` variable indicates where the blueprint is deployed (should be system namespace for positive behavior, something else for negative behavior). It is not modules' namespace.

File: manager/controllers/app/blueprint_controller_unit_test.go [hunk 2]
```diff
@@ -49,6 +49,7 @@ func TestBlueprintReconcile(t *testing.T) {
 	blueprint, err := readBlueprint("../../testdata/blueprint.yaml")
 	g.Expect(err).To(gomega.BeNil(), "Cannot read blueprint file for test")
 	blueprint.Namespace = blueprintNamespace
+	blueprint.Spec.ModulesNamespace = blueprintNamespace
```
Comment (shlomitk1) [inline manager/controllers/app/blueprint_controller_unit_test.go:52]: Same comment.

Comment (laurieaw63) [reply manager/controllers/app/blueprint_controller_unit_test.go]: Yup good catch; changed

File: manager/controllers/app/blueprint_controller_test.go (hunk 1 above)
Comment (laurieaw63) [reply manager/controllers/app/blueprint_controller_test.go]: Yup good catch; changed

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**